prepare/tasks/ner.py

```diff
@@ -3,8 +3,14 @@
 
 add_to_catalog(
     FormTask(
-        inputs=["text", "entity_type"],
-        outputs=["spans_starts", "spans_ends", "text", "labels"],
+        inputs={"text": "str", "entity_type": "str"},
+        outputs={
+            "spans_starts": "List[str]",
```

Comment:
Shouldn't this be List[int]? If not this way right now, then can you open a separate issue for correcting the types?

Reply:
Right, it's like this in `span_labeling` so I guess it'll be the same here